### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: src/components/Search/Search.url.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg version="1.1" id="Search"
```
Comment (friedjoff) [inline src/components/Search/Search.url.svg]: Looks like a duplicate of `./Search.svg` file. Can we reuse the existing file?

File: src/components/TopicLoader/TopicLoader.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ class TopicLoader extends Component {
       }
       this.updateLayers(activeTopic.layers);
 
-      searchService.setSearches(activeTopic.searches);
+      searchService.setSearches(activeTopic.searches || []);
```
Comment (friedjoff) [inline src/components/TopicLoader/TopicLoader.js]:
```suggestion
      searchService.setSearches(activeTopic.searches || {});
```
Not sure if this makes a difference, but searchService expects an object for searches.

File: src/components/Search/Search.url.svg (hunk 1 above)
Comment (oterral) [reply src/components/Search/Search.url.svg]: I guess we could it's just a renaming

Comment (oterral) [reply src/components/Search/Search.url.svg]: Done

**Comment by friedjoff** [comment]
Thanks!

**Pull request merged**